front/app/modules/commercial/insights/admin/containers/Insights/Details/Network/index.tsx

```diff
@@ -45,18 +45,17 @@ import tracks from 'modules/commercial/insights/admin/containers/Insights/tracks
 import { injectIntl, FormattedMessage } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 import messages from '../../messages';
-
 // styles
 import styled from 'styled-components';
 
 type CanvasCustomRenderMode = 'replace' | 'before' | 'after';
 type Node = NodeObject & IInsightsNetworkNode;
 
 const zoomStep = 0.2;
-const chargeStrength = -25;
-const chargeDistanceMax = 80;
-const linkDistance = 50;
-const visibleKeywordLabelScale = 3.5;
+const chargeStrength = -10;
```

Comment:
These are somewhat magic values based on trial and error. We'll have to test the network with a variety of projects and perhaps adjust them but they seem to work ok for now.